packages/as-types/assembly/safeMath.ts

```diff
@@ -68,12 +71,12 @@ export class SafeMath {
    * @throws if the operation results in a number bigger than u64.MAX_VALUE.
    */
   static mul(a: u64, b: u64): u64 {
-    if (a == 0) {
+    if (a == 0 || b == 0) {
```

Comment:
This update is useless: the check of the null value is not for performance, but to divide by a.